### DISCUSSION TITLE
How to create multiple Docker Swarms?

### DISCUSSION THREAD

**Discussion by dushyantRathore** [BretFisher/ama#188] [Q&A]
Hi @BretFisher,
I was wondering if it is possible to create multiple Docker Swarms on a single system? If so, how and what would be the advantages and use cases?

**Comment by BretFisher** [comment]
Not really. It might be possible to do that using the stand-alone SwarmKit utilities, but I've never seen a reason to do so.  It's best to think of a single OS = single Swarm node.
